fortran/test/fetch_content_integration/test_micm_api.F90

```diff
@@ -42,21 +43,22 @@ subroutine test_api()
     real(c_double), parameter     :: GAS_CONSTANT = 8.31446261815324_c_double ! J mol-1 K-1
     integer                       :: i
     
+    config_path = "configs/chapman"
+    solver_type = 1
```

Comment:
I'm a little hesitant to just have an integer determine the solver type. We will eventually want to have many of the solver options be configurable. What about having a struct for solver options, that can be used to specify the vector vs standard matrix, number of grid cells, ODE algorithm, etc.? maybe we could discuss this as a group to come up with a long-term strategy? We could always create a separate issue for this too

Reply:
I agree. I was uncomfortable with using `int` to represent the solver type but only did in favor of fortran. I updated it using enum type and removed `int`. Could you review it?
I like your idea and I agree we can discuss the design as a group.